fdbserver/storageserver.actor.cpp

```diff
@@ -5617,6 +5859,19 @@ void setAvailableStatus(StorageServer* self, KeyRangeRef keys, bool available) {
 		                                           availableKeys.end,
 		                                           endAvailable ? LiteralStringRef("1") : LiteralStringRef("0")));
 	}
+
+	// When a shard is moved out, delete all related checkpoints created for data move.
+	if (!available) {
```

Comment:
It probably makes sense to log this.  I imagine the SRE team will want to know when the space is freed up.

Reply:
Done.